rbac/server/api/proposals.py

```diff
@@ -185,12 +184,12 @@
         proposal_resource["approvers"] = await fetch_relationships(
             table, "task_id", proposal_resource.get("object")
         ).run(conn)
-    elif "users" in table:
-        # approvers needs to be new manager in update manager scenario
-        proposal_resource["approvers"] = [proposal_resource.get("target")]
-    else:
-        user_resource = await fetch_user_resource(conn, proposal_resource.get("object"))
-        proposal_resource["approvers"] = [user_resource.get("manager")]
+    elif table == "next_admins":
```

Comment:
No admins table.  Recommend going back to users

Reply:
Reverted it back to users